internal/pkg/deploy/cloudformation/stack/project.go

```diff
@@ -79,7 +80,13 @@ func (e *ProjectStackConfig) ResourceTemplate(config *ProjectResourcesConfig) (s
 		return "", &ErrTemplateNotFound{templateLocation: projectResourcesTemplatePath, parentErr: err}
 	}
 
-	template, err := template.New("resourcetemplate").Parse(stackSetTemplate)
+	template, err := template.New("resourcetemplate").
+		Funcs(template.FuncMap{
+			"logicalIDSafe": func(logicalID string) string {
+				return strings.ReplaceAll(logicalID, "-", "DASH")
```

Comment:
lol

Reply:
Please do not laugh at my *excellent* code.